lib/iris/experimental/animate.py

```diff
@@ -6,118 +6,32 @@
-        Color scaling values, see :class:`matplotlib.colors.Normalize` for
-        further details. Default values are determined by the min-max across
-        the data set over the entire sequence.
-
-    See :class:`matplotlib.animation.FuncAnimation` for details of other valid
-    keyword arguments.
+    Warnings
+    --------
+    This function is now **disabled**.
 
-    Returns:
-        :class:`~matplotlib.animation.FuncAnimation` object suitable for
-        saving and or plotting.
-
-    For example, to animate along a set of cube slices::
-
-        cube_iter = cubes.slices(('grid_longitude', 'grid_latitude'))
-        ani = animate(cube_iter, qplt.contourf)
-        plt.show()
+    The functionality has been moved to :func:`iris.plot.animate`.
 
     """
-    kwargs.setdefault("interval", 100)
-    coords = kwargs.pop("coords", None)
-
-    if fig is None:
-        fig = plt.gcf()
-
-    def update_animation_iris(i, cubes, vmin, vmax, coords):
-        # Clearing the figure is currently necessary for compatibility with
-        # the iris quickploting module - due to the colorbar.
-        plt.gcf().clf()
-        plot_func(cubes[i], vmin=vmin, vmax=vmax, coords=coords)
-
-    # Turn cube iterator into a list to determine plot ranges.
-    # NOTE: we check that we are not providing a cube as this has a deprecated
-    # iter special method.
-    if hasattr(cube_iterator, "__iter__") and not isinstance(
-        cube_iterator, iris.cube.Cube
-    ):
-        cubes = iris.cube.CubeList(cube_iterator)
-    else:
-        msg = "iterable type object required for animation, {} given".format(
-            type(cube_iterator)
-        )
-        raise TypeError(msg)
-
-    supported = ["iris.plot", "iris.quickplot"]
-    if plot_func.__module__ not in supported:
-        msg = (
-            'Given plotting module "{}" may not be supported, intended '
-            "use: {}."
-        )
-        msg = msg.format(plot_func.__module__, supported)
-        warnings.warn(msg, UserWarning)
-
-    supported = ["contour", "contourf", "pcolor", "pcolormesh"]
-    if plot_func.__name__ not in supported:
-        msg = (
-            'Given plotting function "{}" may not be supported, intended '
-            "use: {}."
-        )
-        msg = msg.format(plot_func.__name__, supported)
-        warnings.warn(msg, UserWarning)
-
-    # Determine plot range.
-    vmin = kwargs.pop("vmin", min([cc.data.min() for cc in cubes]))
-    vmax = kwargs.pop("vmax", max([cc.data.max() for cc in cubes]))
-
-    update = update_animation_iris
-    frames = range(len(cubes))
-
-    return animation.FuncAnimation(
-        fig, update, frames=frames, fargs=(cubes, vmin, vmax, coords), **kwargs
+    msg = (
+        "The function 'iris.experimental.animate.animate()' has been moved, "
+        "and is now at 'iris.plot.animate()'.\n"
+        "Please replace 'iris.experimental.animate.animate' with "
+        "'iris.plot.animate'."
     )
+    raise Exception(msg)
```

Comment:
@trexfeathers Perhaps raise a `DeprecationWarning` instead?

Reply:
I'm all for being more specific, but it seems important to actually exit with an error here to be as clear as possible. What would you think of the below?

```suggestion
    with warnings.simplefilter("error", category=IrisDeprecation):
        iris._deprecation.warn_deprecated(msg)
```

Reply:
TBH it's fine... previous moves of functionality out of experimental have simply raised an `Exception`, so let's just roll with that :+1:

Reply:
Or have the warning and an `Exception` straight after